syntax = "proto3";
option go_package = "./paddle_ocr";

message Image {
  bytes image = 1; // 需要注意的是，图片数据使用 bytes 类型进行传输。在返回图片数据时，通常需要将其转换为 Base64 编码后再传输。
}


message ImageText {
  bytes image = 1;
  string text = 2;  
}

service PaddleOcrService {
  rpc PaddleOcr(Image) returns (ImageText);
}

